base/init.sh

```diff
@@ -5,4 +5,11 @@ cp -r --no-clobber /home_nbuser_default/. /home/$NB_USER/
 # Local testing has no ${NB_PREFIX): placeholder WORKSPACE_BASE_URL is "/workspace"
 export WORKSPACE_BASE_URL=${NB_PREFIX:="/workspace"}
 export NB_USER=${NB_USER}
+if [ -n "${KF_LANG}" ]; then
+  if [ "${KF_LANG}" = "fr" ]; then
+      export LANG="fr_FR.UTF-8" LANGUAGE="fr_FR.UTF-8" LC_ALL="fr_FR.UTF-8"
+  else
+      export LANG="en_US.UTF-8" LANGUAGE="en_US.UTF-8" LC_ALL="en_US.UTF-8"
+  fi
+fi 
```

Comment:
is our use of `KF_LANG` is just `en` or `fr`, or can it have a region on it?  Just making sure `KF_LANG` can't be something like `fr_FR`, which hilariously would then be set to `LANG="en_US.UTF-8"`

Reply:
Asked Wendy to be completely sure, but yes the `KF_LANG` is hardcoded to either `en` or `fr` [in this file](https://github.com/StatCan/jupyter-apis/blob/master/frontend/src/app/resource-form/form-advanced-options/form-advanced-options.component.ts#L16)